pkg/roachprod/cloud/cluster_cloud.go

```diff
@@ -155,9 +179,43 @@ func (c *Cluster) PrintDetails(logger *logger.Logger) {
 	} else {
 		logger.Printf("(no expiration)")
 	}
+	// Align columns left and separate with at least two spaces.
+	tw := tabwriter.NewWriter(os.Stdout, 0, 8, 2, ' ', 0)
```

Comment:
I noticed that we used to write this cluster information on the `logger` passed to this function, but now we're writing straight to stdout.

This means that this information is now printed on TeamCity's "Build Log" instead of the corresponding cluster creation log as before (where I think it's more useful).

Can we revert back to writing this info to the logger?

Reply:
Thanks @herkolategan for creating https://github.com/cockroachdb/cockroach/pull/120118